style.css

```diff
@@ -84,3 +85,615 @@
   left: 0;
   top: 472px;
 }
+
+.workSection {
+  display: flex;
+  flex-direction: column;
+}
+
+.workSection h1 {
+  width: 181px;
+  height: 50px;
+  left: 16px;
+  top: 672px;
+  font-family: 'Playfair Display', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 50px;
+  line-height: 100%;
+  color: #1a2236;
+  position: absolute;
+}
+
+.imgCard {
+  position: absolute;
+  width: 320px;
+  height: 186px;
+  left: 0;
+  top: 754px;
+  border: 0.333333px solid rgba(255, 255, 255, 0.1);
+  margin-top: 5px;
+}
+
+.card {
+  background: #1a2236;
+  position: absolute;
+  width: 320px;
+  height: 156px;
+  left: 0;
+  top: 940px;
+  backdrop-filter: blur(0.5px);
+}
+
+.card h2 {
+  display: flex;
+  justify-content: center;
+  margin-top: 1rem;
+  margin-left: 1rem;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 20px;
+  line-height: 80%;
+  color: #fff;
+  width: 239px;
+  height: 16px;
+  left: 20px;
+  top: 1302px;
+}
+
+.buttons {
+  margin-top: 1rem;
+  margin-left: 1rem;
+
+  /* position: absolute; */
+}
+
+.projectBtn {
+  display: flex;
+  gap: 10px;
+}
+
+.btn1 {
+  width: 69px;
+  height: 20px;
+  left: 30px;
+  top: 998px;
+  border: none;
+  background: #98dede;
+  color: #1a2236;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 10.3334px;
+  line-height: 80%;
+  text-align: center;
+}
+
+.btn2 {
+  width: 87px;
+  height: 20px;
+  left: 99px;
+  top: 992px;
+  background: #98dede;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 10.3334px;
+  line-height: 80%;
+  color: #1a2236;
+  border: none;
+}
+
+.btn3 {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 10.3334px;
+  line-height: 80%;
+  border: none;
+  color: #1a2236;
+  width: 87px;
+  height: 20px;
+  left: 99px;
+  top: 992px;
+  background: #98dede;
+}
+
+.submitBtn {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
```

Comment:
As requested in the project requirements:
![image](https://user-images.githubusercontent.com/42799579/198422542-a5d172e8-0639-43ac-8cfe-9e8bb3c54e18.png)

Kindly add the button effects corresponding to the ones displayed on Figma template:
![image](https://user-images.githubusercontent.com/42799579/198422522-4f7d8f57-1a00-4aad-a76a-ad32ffa4c621.png)
Keep in mind that this project's goal is to improve your ability to follow design guidelines (imagine Microverse as a client requesting a website) 😉. My tip for you: you can add the pointer effect simply with this:
```css
  cursor: pointer;
```
> Just a note: your design looks flawless, great job!

Reply:
![Screenshot 2022-10-28 at 13 41 35](https://user-images.githubusercontent.com/37203742/198590580-9aa3eb65-4a2e-40fa-a378-8da4183afd8a.png)
![Screenshot 2022-10-28 at 13 41 47](https://user-images.githubusercontent.com/37203742/198590582-2aaea50c-828b-4366-a015-21a488143e39.png)
![Screenshot 2022-10-28 at 13 42 04](https://user-images.githubusercontent.com/37203742/198590584-4e61a76c-cf56-4920-81ae-c0d5c494279c.png)
![Screenshot 2022-10-28 at 13 42 14](https://user-images.githubusercontent.com/37203742/198590590-6de7e755-2141-4e88-804c-9cfb864383b0.png)